infrastructure/kubernetes/main.tf

```diff
@@ -39,6 +39,9 @@ locals {
   k8s_client_certificate     = base64decode(data.azurerm_kubernetes_cluster.k8s_cluster.kube_config.0.client_certificate)
   k8s_client_key             = base64decode(data.azurerm_kubernetes_cluster.k8s_cluster.kube_config.0.client_key)
   k8s_cluster_ca_certificate = base64decode(data.azurerm_kubernetes_cluster.k8s_cluster.kube_config.0.cluster_ca_certificate)
+  backend_storage_class      = "azurefile-csi-nfs"
+  backend_storage_size       = "100Gi"
```

Comment:
way oversized, but:
- we may want to sometimes keep temporary files around for inspection (I added a change yesterday that enabled this for staging, and for whatever dev envs via env vars)
- this is using standard locally-redundant storage for now, but if we wanted to use premium later on, 100Gi is (from what I understand) the minimum allocation
- storage is cheap

Reply:
put this into a var, so it's easy to upscale in the future